README.md

```diff
@@ -1058,6 +1060,23 @@ targets:
       fileReplacerStr: release.aar
 ```
 
+**Example (with Kotlin Multiplatform config)**
+```yaml
+targets:
+  - name: maven
+    mavenCliPath: scripts/mvnw.cmd
+    mavenSettingsPath: scripts/settings.xml
+    mavenRepoId: ossrh
+    mavenRepoUrl: https://oss.sonatype.org/service/local/staging/deploy/maven2/
+    android:
+      distDirRegex: /^sentry-android-.*$/
+      fileReplaceeRegex: /\d\.\d\.\d(-SNAPSHOT)?/
+      fileReplacerStr: release.aar
+    kotlinMultiplatform:
+      rootDistDirRegex: /sentry-kotlin-multiplatform-[0-9]+.*$/
+      appleDistDirRegex: /sentry-kotlin-multiplatform-(macos|ios|tvos|watchos).*/
```

Comment:
Do we need the kotlin config _and_ android config? Shouldn't the android config be `false` if we are making a kotlin release?

Reply:
We need to upload multiple artifacts including Android (it's a target platform that we want to support within Kotlin Multiplatform)

This is everything that needs to be uploaded: <img width="585" alt="Screenshot 2022-09-06 at 02 07 37" src="https://user-images.githubusercontent.com/23364143/188595251-d6a850a9-1294-4c3f-aa7a-8446529e457c.png">